skrub/tests/test_clean_null_strings.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from skrub._clean_null_strings import CleanNullStrings
+from skrub._on_each_column import RejectColumn
+
+
+def test_clean_null_strings(df_module):
+    s = df_module.make_column("c", ["a", "b", "   ", "N/A", None])
+    out = CleanNullStrings().fit_transform(s)
+    expected = df_module.make_column("c", ["a", "b", None, None, None])
+    df_module.assert_column_equal(out, expected)
+
+
+def test_reject_non_string_cols(df_module):
+    with pytest.raises(RejectColumn, match=".*does not contain strings"):
+        CleanNullStrings().fit(df_module.example_column)
+    s = df_module.make_column("c", ["a", "b", "   ", "N/A", None])
+    cleaner = CleanNullStrings().fit(s)
+    cleaner.transform(df_module.example_column)
```

Comment:
```suggestion
    assert cleaner.transform(df_module.example_column) is df_module.example_column
```

Asserting that once fitted, the transformer doesn't reject columns anymore

Reply:
it doesn't reject them anymore but it does not necessarily return the original column (if it contained "N/A" that would be replaced)

Reply:
Yes, but since it returns the original column in the test we could check that it's the original col as well

Reply:
Ok I added the `is` check